packages/cli/scripts/install.js

```diff
@@ -98,10 +176,16 @@ module.exports = function (installationDir) {
         .then(answers => {
           const taskQueue = []
           taskQueue.push(tasks.cloneVersion)
+          if (answers.themeName !== 'classic') {
+            taskQueue.push(tasks.cloneTheme)
+          }
           if (answers.installation === options.installation.installer) {
             taskQueue.push(tasks.installDeps)
             taskQueue.push(tasks.runInstaller)
           }
+          if (answers.themeName !== 'classic') {
```

Comment:
These 3 conditions `answers.themeName !== 'classic'` in this file can be removed when default theme will be extracted to own repository `vsf-classic`. Now it just prevents from downloading and configuring default theme which is not needed, because default theme is currently built-in `vue-storefront`.